features/support/hooks.js

```diff
@@ -124,6 +132,33 @@ function cleanSecurity () {
     });
 }
 
+function grantDefaultRoles () {
```

Comment:
this hook is called after `createFirstAdmin`: it's goal is to re-grant all rights to anonymous user, so that we can then call the other hooks and clean-up Kuzzle database after the tests.